### PR TITLE
Fix json error in some python distributions

### PR DESCRIPTION
**Pull request by vn-ki**
Mentioned in #752. ([This comment](https://github.com/mps-youtube/mps-youtube/issues/752#issuecomment-367111112))

### CODE REVIEW

**Comment by jahlborn** [comment]
Another helpful change would be to move the "try:" in _get_metadata_from_lastfm() up to encapsulate the whole method. that way future issues getting metadata won't cause the player to fail.

File: mps_youtube/util.py [hunk 1]
```diff
@@ -518,7 +518,7 @@ def _get_metadata_from_lastfm(artist, track):
 
     metadata = dict()
 
-    data = json.loads(resp.read())
+    data = json.loads(resp.read().decode('utf-8'))
```
Comment (ids1024) [inline mps_youtube/util.py]: It seems like the `HTTPResponse` returned by `urlopen` can be passed to `json.load`. That would probably be better...

Comment (vn-ki) [reply mps_youtube/util.py]: I have made the change.

**Comment by jahlborn** [comment]
fyi, the alternate version of the patch does _not_ fix the original problem (using load(resp) instead of the loads with decode).  i tested the latest patch the it fails.

**Comment by vn-ki** [comment]
@jahlborn Thanks for the update.

@ids1024 Should we switch back to `json.loads` or is there a workaround with `json.load`?

**Comment by ids1024** [comment]
Does anyone know why this is happening? The following code works for me with Python 3.6. Is it an issue with older Python 3 versions?

```python
import urllib.request
import json
print(json.load(urllib.request.urlopen("https://api.github.com/")))
```

For that matter, `json.loads(b"{}")` works fine; it accepts a bytes object.

**Comment by ids1024** [comment]
Okay, it seems it is a Python version issue. 

From the [docs for `json.loads()`](https://docs.python.org/3.6/library/json.html):

> Changed in version 3.6: s can now be of type bytes or bytearray. The input encoding should be UTF-8, UTF-16 or UTF-32.

Changing it back to `json.loads` is probably the only way to be comparable (or at least, no other way is simpler or better).

**Pull request merged**